### PR TITLE
Add support non-English words in docstring arguments

### PR DESCRIPTION
**Pull request by bschnurr**
image

fixes https://github.com/microsoft/pylance-release/issues/4615

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

**Review by debonte — APPROVED** [review]

**Comment by erictraut** [comment]
@bschnurr, looks like the tests are failing for this PR. If you'd like to get this into this week's pyright release, please fix the tests and merge the PR.

**Comment by Aruelius** [comment]
hi @erictraut

maybe the regex can be

```
!line?.endsWith(':') && !line?.endsWith('::') && !!line.match(/^\s*.*?\S+(\s*\(.*?\))*\s*:\s*/g);
```

cause it seems not that important to match the content to right of the `:`.

**Comment by Aruelius** [comment]
Do we have any updates here?

Maybe I can open a new PR if this PR doesn't progress anymore.

**Comment by erictraut** [comment]
@bschnurr, any update on this PR?

File: packages/pyright-internal/src/tests/docStringConversion.test.ts [hunk 1]
```diff
@@ -746,31 +746,6 @@ Returns:
     _testConvertToMarkdown(docstring, markdown);
 });
 
-test('GoogleWithInvalidTypes', () => {
```
Comment (bschnurr) [inline packages/pyright-internal/src/tests/docStringConversion.test.ts:749]: Now that are regex is very relaxed we are keeping the line break in this example. so its no longer a failure case.

**Comment by bschnurr** [comment]
done

**Comment by Aruelius** [comment]
thank you guys very much!

**Pull request merged**